src/client/Microsoft.Identity.Client/AppConfig/ApplicationConfiguration.cs

```diff
@@ -44,6 +44,8 @@ public string ClientVersion
             internal set { _clientVersion = string.IsNullOrWhiteSpace(value) ? DefaultClientVersion : value; }
         }
 
+        public ITelemetryClient[] TelemetryClients { get; internal set; }

```

Comment:
Should maybe initialize this to empty array somewhere, so we don't have to check if it's null before use each time.

Reply:
Initialize to empty? So that `foreach` just jumps over it?

Reply:
SAL also does the initialize to empty as an additional data point.